storage/persistent_storage/persistent_storage_local/ways-to-provision-local-storage.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ways-to-provision-local-storage"]
+= Local storage overview
+include::_attributes/common-attributes.adoc[]
+:context: ways-to-provision-local-storage
+
+toc::[]
+
+You can use any of the following solutions to provision local storage:
+
+* HostPath Provisioner (HPP)
+* Local Storage Operator (LSO)
+* Logical Volume Manager Storage (LVMS)
+
+[WARNING]
+====
+These solutions only support provisioning node-local storage. The workloads are bound to the nodes that provide the storage. If the node becomes unavailable, so does the workload. To maintain workload availability despite node failures, you must ensure storage data replication through active or passive replication mechanisms.
+====
+
+[id="overview-of-hpp-functionality"]
+== Overview of HostPath Provisioner functionality
+
+You can perform the following actions using HostPath Provisioner (HPP):
+
+* Map the host filesystem paths to storage classes for provisioning local storage.
+* Statically create storage classes to configure filesystem paths on a node for storage consumption.
+* Statically provision Persistent Volumes (PVs) based on the storage class.
+* Create workloads and PersistentVolumeClaims (PVCs) while being aware of the underlying storage topology.
+
+[NOTE]
+====
+HPP is available in upstream Kubernetes. However, it is not recommended to use HPP from upstream Kubernetes.
+====
+
+[id="overview-of-lso-functionality"]
+== Overview of  Local Storage Operator functionality
+
+You can perform the following actions using Local Storage Operator (LSO):
+
+* Assign the storage devices (disks or partitions) to the storage classes without modifying the device configuration.
+* Statically provision PVs and storage classes by configuring the `LocalVolume` custom resource (CR).
```

Comment:
LSO providers both `LocalVolume` or `LocalVolumeSet` as CR for provisioning.
```suggestion
* Statically provision PVs and storage classes by configuring the `LocalVolume` or `LocalVolumeSet` custom resource (CR).
```

Reply:
@sr1kar99 @gcharot 
Sorry, I did not realize that. So it's okay to me if not mentioning `LocalVolumeSet` or adding the TechPreview as an annotation.

Reply:
As the core point "Statically provision PVs and storage classes" is already mentioned, excluding the `LocalVolumeSet` as suggested.
Thanks!